xAPI-Data.md

```diff
@@ -1451,52 +1457,41 @@ State resources relating to the same learning experience so that all data record
 A map of the types of learning activity context that this Statement is related to.
 
 ###### Rationale
-Many Statements do not just involve one Object Activity that is the focus,
-but relate to other contextually relevant Activities. "Context Activities" allow for 
-these related Activities to be represented in a structured manner.
+Many Statements do not just involve one (Object) Activity that is the focus, but relate to other contextually relevant 
+Activities. "Context Activities" allow for these related Activities to be represented in a structured manner.
 
 ###### Details
 There are four valid context types. All, any or none of these MAY be used in a given Statement:
 
-1. __Parent__: an Activity with a direct relation to the Activity
-which is the Object of the Statement. In almost all cases there
-is only one sensible parent or none, not multiple.
-For example: a Statement about a quiz question would have the quiz
+1. __Parent__: an Activity with a direct relation to the Activity which is the Object of the Statement. In almost all cases 
+there is only one sensible parent or none, not multiple. For example: a Statement about a quiz question would have the quiz 
 as its parent Activity.
  
-2. __Grouping__: an Activity with an indirect relation to the Activity
-which is the Object of the Statement.
-For example: a course that is part of a qualification. The course
-has several classes. The course relates to a class as the parent,
+2. __Grouping__: an Activity with an indirect relation to the Activity which is the Object of the Statement. For example: 
+a course that is part of a qualification. The course has several classes. The course relates to a class as the parent, 
 the qualification relates to the class as the grouping.
 
-3. __Category__: an Activity used to categorize the Statement.
-"Tags" would be a synonym. Category SHOULD be used to indicate
-a "profile" of xAPI behaviors, as well as other categorizations.
-For example: Anna attempts a biology exam, and the Statement is
-tracked using the cmi5 profile. The Statement's Activity refers
-to the exam, and the category is the cmi5 profile.
+3. __Category__: an Activity used to categorize the Statement.  "Tags" would be a synonym. Category SHOULD be used to 
```

Comment:
There's a few other instances of double spaces in this PR, some of which already exist. If you use Chrome's find feature on this page you'll be able to spot them.